GliaWidgets/Sources/ViewModel/Chat/ChatViewModel.swift

```diff
@@ -495,6 +490,30 @@ extension ChatViewModel {
         messageText = ""
     }
 
+    func handleSendMessageError(_ error: CoreSdkClient.GliaCoreError) {
+        switch error.error {
+        case let authenticationError as CoreSdkClient.Authentication.Error:
+            switch authenticationError {
+            case .expiredAccessToken:
+                break
+            default:
+                environment.log.prefixed(Self.self).info("Message send exception")
+                environment.log.prefixed(Self.self).info("Show Unexpected error Dialog")
+                self.showAlert(
+                    with: self.alertConfiguration.unexpectedError,
+                    dismissed: nil
+                )
+            }
+        default:
+            environment.log.prefixed(Self.self).info("Message send exception")
+            environment.log.prefixed(Self.self).info("Show Unexpected error Dialog")
+            self.showAlert(
+                with: self.alertConfiguration.unexpectedError,
+                dismissed: nil
+            )
+        }
```

Comment:
could we simplify this switch? perhaps to something like this?

```swift
        switch error.error {
        case let authenticationError as CoreSdkClient.Authentication.Error where authenticationError == .expiredAccessToken:
                break
        default:
            environment.log.prefixed(Self.self).info("Message send exception")
            environment.log.prefixed(Self.self).info("Show Unexpected error Dialog")
            self.showAlert(
                with: self.alertConfiguration.unexpectedError,
                dismissed: nil
            )
        }
```

Reply:
fixed [6a2873a](https://github.com/salemove/ios-sdk-widgets/pull/959/commits/6a2873a734ab1214de900b20b3908782b8fc09b3)

Reply:
Can't and will not merge before CoreSDK release because `expiredAccessToken` has an associated value. However for testing purposes I had Core connected directly and this issue didn't emerge